### PR TITLE
Markdown file for OHBM tutorial materials and add to menu.

### PR DESCRIPTION
**Pull request by kristinazvolanek**
Closes #

## Proposed Changes

  -
  -
  -

## Change Type

- [ ] `bugfix`
- [ ] `refactoring`
- [ ] `infrastructure`
- [ ] `documentation`
- [ ] `other`

## Checklist before review

- [ ] I added everything I wanted to add to this PR.
- [ ] \[Documentation only\] I built the docs locally.
- [ ] My contribution is harmonious with the rest of the code: I'm not introducing repetitions.
- [ ] My code respects the adopted style, especially linting conventions.
- [ ] The title of this PR is explanatory on its own, enough to be understood as part of a changelog.
- [ ] I added or indicated the right labels.

- [ ] I added information regarding the timeline of completion for this PR.

- [ ] Please, comment on my PR while it's a draft and give me feedback on the development!

### CODE REVIEW

**Comment by kristinazvolanek** [comment]
@smoia @m-miedema I think I made a markdown page to add our links to the physiopy site... please review to see if I missed any important information!

**Comment by smoia** [comment]
@kristinazvolanek did you try to build this locally?
@m-miedema can you have a look?
@goodalse2019 sorry to bother you, but could you check this out too?

File: mkdocs.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ nav:
       - Other Tools: community/other_tools.md
       - Code of Conduct: community/CODE_OF_CONDUCT.md
       - Team: community/team.md
+    -OHBM 2023 Tutorials: ohbm23_tutorials.md
```
Comment (goodalse2019) [inline mkdocs.yml:26]: You just need to add a space after the dash and before OHBM for it to correctly catch the section!

**Review by goodalse2019 — CHANGES_REQUESTED** [review]
If the screenshot I shared with Kristina looks good and then just that space addition in mkdocs this should be good to go!

**Review by smoia — APPROVED** [review]

**Pull request merged**